src/main/resources/application.yml

```diff
@@ -1,3 +1,9 @@
+spring:
+  flyway:
+    enabled: true
+    baselineOnMigrate: true
```

Comment:
`Be careful when enabling this as it removes the safety net that ensures Flyway does not migrate the wrong database in case of a configuration mistake!`

Reply:
Nice catch fikser det!